description/en.txt

```diff
@@ -1,10 +1,12 @@
-AntennaPod is a podcast manager that gives you instant access to millions of free and paid podcasts, from independent podcasters to large publishing houses such as the BBC, NPR and CNN. Add, import and export their feeds hassle-free using the iTunes podcast database, OPML files or simple RSS URLs. Save effort, battery power and mobile data usage with powerful automation controls for downloading episodes (specify times, intervals and WiFi networks) and deleting episodes (based your favourites and delay settings).<br>
+Easy-to-use, flexible and open-source podcast manager and player
+
+AntennaPod is a podcast manager and player that gives you instant access to millions of free and paid podcasts, from independent podcasters to large publishing houses such as the BBC, NPR and CNN. Add, import and export their feeds hassle-free using the iTunes podcast database, OPML files or simple RSS URLs. Save effort, battery power and mobile data usage with powerful automation controls for downloading episodes (specify times, intervals and WiFi networks) and deleting episodes (based your favourites and delay settings).<br>
 But most importantly: Download, stream or queue episodes and enjoy them the way you like with adjustable playback speeds, chapter support and a sleep timer. You can even show your love to the content creators  with our Flattr integration.
 
 Made by podcast-enthousiast, AntennaPod is free in all senses of the word: open source, no costs, no ads.
 
 <b>All features:</b><br>
-IMPORT, MANAGE AND PLAY<br>
+IMPORT, ORGANISE AND PLAY<br>
```

Comment:
Since we're usually targeting US english this should be 'ORGANIZE': http://grammarist.com/spelling/organise-organize/

Also, can we just use normal title case instead of all caps for them?